config/permissions.php

```diff
@@ -125,18 +135,24 @@
         '/settings.php')
 );
 
-/** Admin permissions array
+/** Extra permissions array
  *
  * It contains a multiple-level array with the groups and the
  * pages they are allowed to act on as admin. In every element of
  * the array, the key is the name of the group and the content is
  * another array with the urls the group is allowed to open.
  * For example: "clients" => array("/xptracker-summary.php")
  */
-$adminPermissions = array(
+$extraPermissions = array(
     "admin" => array(
         //user tasks report
-        "/userTasksReport.php", "/services/getTasksFiltered.php",
+        "/userTasksReport.php", "/services/getTasksFiltered.php", "/viewUserDetails.php",
         //users management
-        "/viewUsers.php"),
-);
+        "/viewUsers.php", "/services/getUserProjectCustomerReportJsonService.php"
```

Comment:
Same here! An admin needs to be a manager too, so you don't need to repeat `viewUserDetails` and `getUserProjectCustomerReportJsonService`.